### DISCUSSION TITLE
读取数据时，程序在一直在core.data.dataloader._RepeatSampler（）这个地方无法跳出

### DISCUSSION THREAD

**Discussion by jtkullo** [RL-VIG/LibFewShot#103] [Q&A]
读取数据到如下情况时卡住：
[01/18/24 15:46:19] INFO     load 40 train image with 4 label.    trainer.py:372
通过debug发现在core.data.dataloader._RepeatSampler（）这个地方几乎死循环，程序在运行，但无法继续下一步。
望指点，感谢。

**Comment by VincenDen** [comment]
这边看起来您的label只有4个，fewshot任务的设定一般为5类，所以您需要检查一下您的配置文件

**Reply by jtkullo** [reply]
感谢您的回复，我已经对配置文件进行了合理地修改，现在程序可以正常运行，再次感谢。
